Modules/_hashopenssl.c

```diff
@@ -2109,6 +2109,7 @@ hashlib_init_constructors(PyObject *module)
     if (PyModule_AddObjectRef(module, "_constructors", proxy) < 0) {
         return -1;
     }
+    Py_DECREF(proxy);
```

Comment:
Decref in case `PyModule_AddObjectRef` fails, no?

```suggestion
    int rc = PyModule_AddObjectRef(module, "_constructors", proxy);
    Py_DECREF(proxy);
    if (rc < 0) {
        return -1;
    }
```

Reply:
FYI, fixed in #26079